src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -293,9 +295,16 @@ const styles = StyleSheet.create({
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
-    backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
-    marginRight: 8
+    backgroundColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
+    marginRight: 8,
+    alignContent: 'center',
+    justifyContent: 'center'
   }),
+  profileHashtag: {
+    color: COLORS.white,
+    textAlign: 'center',
+    fontSize: normalizeFontSize(24)
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundColor` property in the `image` style has been updated to handle both 'incognito' and 'signed' follow types. A new style named `profileHashtag` has been introduced which is used for the `Text` component inside the `View` component when no image URL is provided. These changes are consistent with the updates made in the image rendering logic.
```diff
- backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
+ backgroundColor: followType === 'incognito' ? COLORS.anon_secondary : COLORS.signed_primary,
+ marginRight: 8,
+ alignContent: 'center',
+ justifyContent: 'center'
+ }),
+ profileHashtag: {
+   color: COLORS.white,
+   textAlign: 'center',
+   fontSize: normalizeFontSize(24)
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->